jtl/functions.go

```diff
@@ -1009,6 +1015,30 @@ func fnProp(ctx Context, doc *JDoc, params []string) interface{} {
 	return doc.ParseExpression(ctx, props[extractStringParam(params[0])])
 }
 
+func fnPropExists(ctx Context, doc *JDoc, params []string) interface{} {
+	stats := ctx.Value(EelTotalStats).(*ServiceStats)
+	if params == nil || len(params) != 1 {
+		ctx.Log().Error("error_type", "func_propExists", "op", "propExists", "cause", "wrong_number_of_parameters", "params", params)
+		stats.IncErrors()
+		AddError(ctx, SyntaxError{fmt.Sprintf("wrong number of parameters in call to unquote function"), "unquote", params})
+		return ""
+	}
+
+	ctx.Log().Debug("op", "propExists", "params", params[0])
+
+	cp := GetCustomProperties(ctx)
+	if cp != nil {
```

Comment:
Looks like GetCustomProperties will never return nil

Reply:
Yeah, I think it is fine for now.  I basically copy that code from the prop function.

Reply:
okay